latex2lambda.py

```diff
@@ -0,0 +1,48 @@
+from latex2sympy2 import latex2sympy
+from sympy import lambdify
+
+
+class TexWithoutArgumentsError(Exception):
+    def __init__(self, msg: str):
+        super().__init__(msg)
+
+
+def latex2lambda(
+    tex: str,
+    variable_values={},
+    args=None,
+    modules=None,
+    printer=None,
+    use_imps=True,
+    dummify=False,
+):
+    if args is None:
+        try:
+            """
+            Trying to parse a function of type "f(...) = <...>" with the function itself on <...>
+            """
+            var_tex, tex = tex.replace(" ", "").split("=")
+            args = var_tex.split("(")[1].replace(")", "").split(",")
```

Comment:
Last, but not least, since you're doing input validation on this lines and want to still catch some low-hanging fruits without much effort, you probably want to:
1. cover other types of white spaces, i.e., tabs (`\t`), newline marker (from multiple systems!, `\r\n` vs `\n`);
2. cover for the use-case of multiple equal signs -> and maybe just throw an exception here (you'd have to remove the `try...catch` block, and actually do some input validation before raising exceptions)
3. catch some edge cases with commas on the argument list